src/WebJobs.Extensions.Http/Extensions/HttpRequestExtensions.cs

```diff
@@ -48,20 +49,32 @@ public static async Task<string> ReadAsStringAsync(this HttpRequest request)
             return request.Query.ToDictionary(p => p.Key, p => p.Value.Last());
         }
 
-        public static void AddIdentity(this HttpRequest request, ClaimsIdentity identity, string headerName)
+        public static ClaimsIdentity GetEasyAuthIdentity(this HttpRequest request)
```

Comment:
I would prefer we don't use "Easy Auth" in public APIs. How about using "AppService" instead?